### PR TITLE
GCB Wait on VCR status, retry failed tests in RECORDING mode

### PR DESCRIPTION
**Pull request by slevenick**
**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

File: .ci/containers/terraform-vcr-tester/check_membership.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+PR_NUMBER=$1
+
+USER=$(curl -H "Authorization: token ${GITHUB_TOKEN}" \
+  "https://api.github.com/repos/GoogleCloudPlatform/magic-modules/issues/${PR_NUMBER}" | jq -r .user.login)
+
+# Only run tests for safe users
+if $(echo $USER | fgrep -wq -e ndmckinley -e danawillow -e emilymye -e megan07 -e paddycarver -e rambleraptor -e SirGitsalot -e slevenick -e c2thorn -e rileykarson); then
```
Comment (ndmckinley) [inline .ci/containers/terraform-vcr-tester/check_membership.sh:10]: Can we have this list in a shared file someplace?

**Review by ndmckinley — APPROVED** [review]

File: .ci/containers/terraform-vcr-tester/check_membership.sh (hunk 1 above)
Comment (slevenick) [reply .ci/containers/terraform-vcr-tester/check_membership.sh:10]: Yeah, merging as-is to allow builds to pass that reference the new trigger. Will come back and refactor

**Pull request merged**